packages/at_persistence_secondary_server/lib/src/keystore/hive_keystore.dart

```diff
@@ -328,6 +328,10 @@ class HiveKeystore implements SecondaryKeyStore<String, AtData?, AtMetaData?> {
       // On deleting the key, remove it from the expiryKeyCache.
       _expiryKeysCache.remove(key);
       if (skipCommit) {
+        CommitEntry? commitEntry = _commitLog.getLatestCommitEntry(key);
```

Comment:
Please add docs on why we are deleting a key from commit log

Reply:
Updated in [531e0e1](https://github.com/atsign-foundation/at_server/pull/2076/commits/531e0e16eb5ee3488ca44e65308baf1baf9d3df1)[531e0e1](https://github.com/atsign-foundation/at_server/pull/2076/commits/531e0e16eb5ee3488ca44e65308baf1baf9d3df1)